docs/topics/i18n.rst

```diff
@@ -2,138 +2,38 @@
 Internationalisation
 ####################
 
+django CMS excels in its multilingual support, and can be configured to handle a vast range of different requirements. Its
+behaviour is flexible and can be controlled at a granular level in :setting:`CMS_LANGUAGES`. Other :ref:`i18n_l10n_reference` settings offer further control.  
 
-*****************
-Multilingual URLs
-*****************
+See :ref:`multilingual_support_how_to` on how to set up a multilingual django CMS project.
 
-If you use more than one language, django CMS urls, *including the admin URLS*, need to be
-referenced via :func:`~django.conf.urls.i18n.i18n_patterns`. For more information about this see
+****
+URLs
+****
+
+Multilingual URLs require the use of :func:`~django.conf.urls.i18n.i18n_patterns`. See For more information about this see
```

Comment:
"See For more information about this see" should be "For more information about this see"